release/src/main/scripts/build_release_candidate.sh

```diff
@@ -90,6 +91,14 @@ if [[ $confirmation != "y" ]]; then
   exit
 fi
 
+RC_TAG="v${RELEASE}-RC${RC_NUM}"
```

Comment:
Let's add a new section header (since this is not a part of "Checking Environment Variables").

Reply:
Moved it up to env var setup.